app/models/conference_preference.rb

```diff
@@ -9,13 +8,11 @@ class ConferencePreference < ActiveRecord::Base
 
   private
 
-    def terms_accepted?
+    def terms
       conference = first_conference_id.present? || second_conference_id.present?
       terms_checkbox = terms_of_ticket == '1' && terms_of_travel == '1'
       first_case = conference.present? && terms_checkbox.present?
```

Comment:
Using "present?" for something that is already a boolean looks a bit confusing for me...
```ruby
first case = conference_chosen && terms_checked
```
would do the same, but more expressive, what do you think?